tests/swap_test.go

```diff
@@ -132,7 +134,7 @@ var _ = Describe("[Serial][sig-compute]SwapTest", Serial, decorators.SigCompute,
 	}
 
 	getAffinityForTargetNode := func(targetNode *v1.Node) (nodeAffinity *v1.Affinity, err error) {
-		nodeAffinityRuleForVmiToFill, err := affinityToMigrateFromSourceToTargetAndBack(targetNode, targetNode)
+		nodeAffinityRuleForVmiToFill, err := migration_utils.AffinityToMigrateFromSourceToTargetAndBack(targetNode, targetNode)
```

Comment:
nit: what the func does is not clear to me by reading it
```suggestion
		nodeAffinityRuleForVmiToFill, err := migration_utils.CreateNodeAffinityRuleToMigrateFromSourceToTargetAndBack(targetNode, targetNode)
```